newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -261,6 +261,11 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 				Description:  "The amount of time (in seconds) to wait before considering the signal expired.  Must be in the range of 30 to 172800 (inclusive)",
 				ValidateFunc: validation.IntBetween(30, 172800),
 			},
+			"ignore_on_expected_termination": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				Description:  "Do not open a loss of signal on an expected termination.",
```

Comment:
Nit: `Whether to ignore expected termination of a signal when considering whether to create a loss of signal incident.` Aligns more closely with AGQL description.